src/components/Footer.js

```diff
@@ -88,6 +89,9 @@ export default function Footer(children) {
             <a href="/Library/">
               <List />
             </a>
+            <a href="/Fetch/">
+              <Fetch />
```

Comment:
I would suggest that you call this "FetchIcon" to make it more clear, that there is not actually a fetch-functionality hidden behind this component